client/mysqltest.cc

```diff
@@ -7742,7 +7742,7 @@ void append_info(DYNAMIC_STRING *ds, ulonglong affected_rows,
                  const char *info)
 {
   char buf[40], buff2[21];
-  sprintf(buf,"affected rows: %s\n", llstr(affected_rows, buff2));
+  snprintf(buf, sizeof(buf), "affected rows: %s\n", llstr(affected_rows, buff2));
```

Comment:
I think 40 should be enough, the largest length `llstr()` could return is 20 characters.